### ISSUE TITLE
Collection row order

### ISSUE THREAD

**Issue by mathieu-fontaine** [jamalex/notion-py#6]
Is it possible to specify an order for the rows of a collection?

**Comment by jamalex** [comment]
In principle, yes! It needs to be done via a CollectionView, however (just based on how queries work in the backend), something like this:

```
cv = client.get_collection_view("https://www.notion.so/learningequality/e91d78e05fcc445a99ebbabe5c54428a?v=aafe1755187343d2a2b5b03d36f2a6c4")
rows = cv.build_query(sort=[{"property": "title", "direction": "ascending"}]).execute()
```

However, it looks like there's a bug in my code that will prevent that from working. I'll submit a fix shortly, which will go into 0.0.21.

**Comment by jamalex** [comment]
Fixed in https://github.com/jamalex/notion-py/pull/23, which will shortly be released as 0.0.21!

**Comment by jamalex** [comment]
Version 0.0.21 has now been [released to PyPI](https://pypi.org/project/notion/0.0.21/)! Please give it a try and see how it works for you. Thanks for reporting issues!
